packages/core/src/linker/ng_module_factory_loader.ts

```diff
@@ -17,23 +20,50 @@ export abstract class NgModuleFactoryLoader {
   abstract load(path: string): Promise<NgModuleFactory<any>>;
 }
 
-let moduleFactories = new Map<string, NgModuleFactory<any>>();
+/**
+ * Map of module-id to the coresponting NgModule.
+ * - In pre Ivy we track NgModuleFactory,
+ * - In post Ivy we track the NgModuleType
+ */
+let modules = new Map<string, NgModuleFactory<any>|NgModuleType>();
```

Comment:
No longer reassigned.
```suggestion
const modules = new Map<string, NgModuleFactory<any>|NgModuleType>();
```